### PR TITLE
added contribution.md file

### PR DESCRIPTION
**Pull request by deveshio**
Added contribution.md files #5

### CODE REVIEW

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- [Contribution Process](#contribution-process)
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [License](#license)
+
+## Getting Started
+
+1. Fork the repository.
+2. Clone the forked repository to your local machine:
+    ```bash
+    git clone https://github.com/your-username/[project-name].git
```
Comment (arpitghura) [inline CONTRIBUTION.md]:
```suggestion
    git clone https://github.com/<your-username>/NotesJar.git 
```
Adding project name and also add a line to change `<your-username>` with their own username.

File: CONTRIBUTION.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    git clone https://github.com/your-username/[project-name].git 
+    ```
+3. Create a new branch for your feature or bug fix:
+    ```bash
+    git checkout -b feature-name
+    ```
+4. Make your changes and commit them with a clear and concise message:
+    ```bash
+    git add .
+    git commit -m "Your clear and concise message"
```
Comment (arpitghura) [inline CONTRIBUTION.md]: Add this blog link on `How to write Better Git Commit Messages` by `FreeCodeCamp` along with the credits.

Link to the post: [https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/)

File: CONTRIBUTION.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    ```
+5. Push your changes to your fork:
+    ```bash
+    git push origin feature-name
+    ```
+6. Open a pull request, referencing any relevant issues.
```
Comment (arpitghura) [inline CONTRIBUTION.md]: Remove the word `any`.

Mention these:

The PR should contain about:
- the problem or issue tag
- the solution approach you have made
- any big file changes
- any bug/issue that can arise from the changes.

File: CONTRIBUTION.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+3. Include relevant tests for your changes.
+4. Provide a clear and concise description of your pull request.
+
+## License
+
+By contributing to this project, you agree that your contributions will be licensed under the [Project License].
+
+---
+
+Thank you for contributing to [Project Name]! 🚀
```
Comment (arpitghura) [inline CONTRIBUTION.md]: Change the [Project Name] to NotesJar. 
and [Project Licence] to MIT.

**Review by arpitghura — CHANGES_REQUESTED** [review]
Make these required changes.

**Comment by deveshio** [comment]
I have update the contribution.md file

**Review by arpitghura — APPROVED** [review]
Looks Good to Me.

**Comment by arpitghura** [comment]
One thing to change is the name of the file, the file name should be `CONTRIBUTING.md`

**Comment by deveshio** [comment]
Sir ,I request you please check your requirements and merge  this update with your branch .
i have update the name

**Review by arpitghura — APPROVED** [review]
LGTM

**Pull request merged**